hack/kind/kind.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+KIND_VERSION="0.17.0"
+
+__dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+function check() {
+	if [ "$(kind --version)" == "kind version $KIND_VERSION" ]; then
```

Comment:
Why? please add a comment with the reason

Also, this is a bit intrusive, it would (and it did in my case) swap the original's system kind with version 0.17.0

Can we at least prompt the user to accept this or not?

Reply:
Agreed

Reply:
Do you think writing it in the docs will suffice ?

Reply:
I don't think so, users will most likely run this script without giving it much thought, and they can end up with a kind binary different from what they had in the host